daemon/create.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 // ContainerCreate takes configs and creates a container.
-func (daemon *Daemon) ContainerCreate(name string, config *runconfig.Config, hostConfig *runconfig.HostConfig, adjustCPUShares bool) (types.ContainerCreateResponse, error) {
+func (daemon *Daemon) ContainerCreate(name string, config *runconfig.Config, hostConfig *runconfig.HostConfig, adjustCPUShares bool, fromBuilder bool) (types.ContainerCreateResponse, error) {
```

Comment:
No, you know create private func with bool arg and two public wrappers without. Both solution will fit my view - config struct instead of arguments or such wrappers.